README.md

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+POLUS is a data processing package integrated in the workflow of the machine-learned force field FFLUX. The package mainly works on
+output from molecular dynamics simulators and electronic structure calculators. It handles a list of input files, including .csv, .txt,
+and .xyz files. 
+
+POLUS performs various tasks, such as: 
+- Extraction of representative subsets of geometries based on structural diversity.
+- Correction of atomic IQA energies following a penalised stakeholder master equation.
+- Identification of spatially equivalent atoms based on the distribution of local properties along a dynamic path.
+- Stratification of datasets based on the distribution of both atomic and molecular properties.
+
+# Instalation
+
+POLUS can be easily installed by following these steps:
+
+1. Download or clone a copy of the copy. For the moment, we have only
+made available the development version called "polus-dev".
+
+2. Move inside the downloaded/cloned folder: ```cd polus-dev```
+
+3. ```pip install .```
+
+We recommend that the package be installed in a dedicated virtual environment.
+Assuming the name of the environment is "polus", it is easy to create one following
+one of the two solutions:
+
+-  ```python -m venv ~/.venv/polus```
+
+OR
+
+- ```conda create --name polus```
+
+In either case, your virtual environment can be activated as follows:
+
+1.  ```source ~/.venv/polus/bin/activate```
```

Comment:
I wonder if it's a bit confusing to put the activation instructions separately from the creation instructions?  Might it be a little clearer if they were grouped together, for people who are really new to virtual environments?  A possible approach is shown below.

____
Assuming the name of the environment is "polus", it is easy to create one following one of two solutions:

Using the built-in Python venv solution, create the virtual environment using:

    python -m venv ~/.venv/polus

and activate the virtual environment using:

    source ~/.venv/polus/bin/activate

For the Conda venv solution, create the virtual environment using:

    conda create --name polus

and then activate it using:

    conda activate polus
    
____

Just a suggestion!  Ignore if not helpful.

Reply:
Hello! This is definitely an important suggestion, thank you. I will edit file tonight and I hope you’ll merge it soon afterwards. Thanks again

Reply:
Fixed it